content/en/user-guide/aws/opensearch/index.md

```diff
@@ -185,56 +82,48 @@ $ curl -s http://my-domain.us-east-1.opensearch.localhost.localstack.cloud:4566/
 
-#### Custom Endpoints
+## Custom Endpoints
 
-LocalStack allows you to set arbitrary custom endpoints for your clusters in the domain endpoint options.
-This can be used to overwrite the behavior of the endpoint strategies described above.
-You can also choose custom domains, however it is important to add the edge port (`80`/`443` or by default `4566`).
+LocalStack allows you to define arbitrary endpoints for your clusters within the domain endpoint options. This functionality can be used to overwrite the behavior of the aforementioned endpoint strategies. Moreover, you can opt for custom domains, though it's important to incorporate the edge port (80/443, or the default 4566).
+
+Run the following command to create a new OpenSearch domain with a custom endpoint:
 
 {{< command >}}
 $ awslocal opensearch create-domain --domain-name my-domain \
       --domain-endpoint-options '{ "CustomEndpoint": "http://localhost:4566/my-custom-endpoint", "CustomEndpointEnabled": true }'
 {{< / command >}}
 
-Once the domain processing is complete, you can access the cluster:
+After the domain processing is complete, you can access the cluster using the custom endpoint:
 
 {{< command >}}
 $ curl http://localhost:4566/my-custom-endpoint/_cluster/health
 {{< / command >}}
 
+## Re-using a single cluster instance
 
-### Re-using a single cluster instance
-
-In some cases, you may not want to create a new cluster instance for each domain,
-for example when you are only interested in testing API interactions instead of actual OpenSearch functionality.
-In this case, you can set `OPENSEARCH_MULTI_CLUSTER=0`, which will multiplex all domains to the same cluster.
-This can however lead to unexpected behavior when persisting data into OpenSearch, or creating clusters with different versions, so we do not recommend it.
+In certain scenarios, creating a distinct cluster instance for each domain might not align with your use-case. For example, if your focus is solely on testing API interactions rather than actual OpenSearch functionality, individual clusters might be excessive. In such situations, the option to set `OPENSEARCH_MULTI_CLUSTER=0` exists, allowing all domains to be funneled into a single cluster instance.
 
+However, it's important to be aware that it can introduce unexpected complications. This is particularly true when dealing with data persistence within OpenSearch or when working with clusters of varying versions. As a result, we advise caution when considering this approach and generally recommend against it.
 
-### Storage Layout
+## Storage Layout
 
 OpenSearch will be organized in your state directory as follows:
 
-```plaintext
+```bash
 localstack@machine % tree -L 4 ./volume/state
```

Comment:
This section should use the command shortcode and not show the `localstack@machine %` prefix.